### PR TITLE
Enable overriding multiple contexts inline

### PR DESCRIPTION
**Pull request by osherdp**
Let the user give more than one context/job names to override, e.g.:
```
/override this-job that-other-job
```

This should make it similar to the ``/test`` command that is able to trigger multiple jobs at once, and should be more intuitive to users.

### CODE REVIEW

**Comment by osherdp** [comment]
/hold
waiting until reviewers take a close look at that regex change

File: prow/plugins/override/override.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ import (
 const pluginName = "override"
 
 var (
-	overrideRe = regexp.MustCompile(`(?mi)^/override( (.+?)\s*)?$`)
+	overrideRe = regexp.MustCompile(`(?mi)^/override( (.+)\s*)*$`)
```
Comment (matthyx) [inline prow/plugins/override/override.go:44]: Indeed I'm not convinced by your regex... few questions (without playing with it):

- why did you drop the lazy match `.+?` for  a greedy
- I think you need more parenthesis because the outer capture group repeats now, so it should only match the last iteration

**Review by matthyx — CHANGES_REQUESTED** [review]
revert the regex change, or use groups to treat multiple contexts

File: prow/plugins/override/override.go (hunk 1 above)
Comment (osherdp) [reply prow/plugins/override/override.go]: * Indeed, with the usage of ``strings.Fields``` I don't need this regex change is needed at all. Reverted
* All tests still passing, so please let me know if there are any edge cases that are missing tests and I can add them in this PR or another

Comment (osherdp) [reply prow/plugins/override/override.go]: In the end I think it can even be simplified into ```(?mi)^/override( (.+))?$```, because we'll strip all whitespaces either way with the ``strings.Fields``

**Comment by matthyx** [comment]
/unhold
@chaodaiG I think it's fine

**Review by chaodaiG — APPROVED** [review]

**Comment by upodroid** [comment]
Hi

This change broke the feature I implemented in https://github.com/kubernetes/test-infra/pull/27017

Proof: https://github.com/knative/serving/pull/13292#issuecomment-1244519995

@chaodaiG  Can we revert this change?

**Comment by chaodaiG** [comment]
> Hi
> 
> This change broke the feature I implemented in #27017
> 
> Proof: [knative/serving#13292 (comment)](https://github.com/knative/serving/pull/13292#issuecomment-1244519995)
> 
> @chaodaiG Can we revert this change?

To be more specific, what's broken here is:
```
/override style / Lint
```

Note that the GH action name `style / Lint` contains empty spaces, which is not a common use case in Prow. Please consider fixing forward instead, two options in my mind:
- Rename GH actions by removing spaces
- Supporting quotes
